core/DataArray.php

```diff
@@ -424,7 +424,7 @@ public static function isRowActions($row)
      *
      * @return \Piwik\DataTable
      */
-    public function asDataTable()
+    public function asDataTable($arrayHasSummaryRow = false)
```

Comment:
This will break things for our premium features which partially overwrite this method. Might be better to create a new method for this ?